examples/twittermap/web/public/javascripts/common/module-manager.js

```diff
@@ -0,0 +1,81 @@
+angular.module("cloudberry.common")
+  .service("moduleManager", function() {
+
+    var moduleManager = {
+
+      // Built-in events
+      EVENT: {
+        CLICK_SEARCH_BUTTON: "clickSearchButtonEvent",
+        CHANGE_TIME_SERIES_RANGE: "changeTimeSeriesRangeEvent",
+        CHANGE_ZOOM_LEVEL: "changeZoomLevelEvent",
+        CHANGE_REGION_BY_DRAG: "changeRegionByDragEvent",
+        CHANGE_MAP_TYPE: "changeMapTypeEvent"
+      },
```

Comment:
Can't we remove "click..." string?

Reply:
I tried, not able to remove them. And after some investigation, to implement an enumeration object is not trivial, so I gave up the idea to make it close, now this object is only for the sake of readability of code, without any constraint for users.

Reply:
Got it. I just checked too. Thanks for the investigation. Can we just use a simple constant after the event type such as "CHANGE_MAP_TYPE: 4" since the string part isn't really necessary.